<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>online_datasaving_ui</class>
 <widget class="QDialog" name="online_datasaving_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data saving</string>
  </property>
  <widget class="QLabel" name="lbl_filepath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>File path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_filepath">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start_stop">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start saving</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time_elapsed">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTimeEdit" name="te_maxtime">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_maxtime">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop after this time</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
